.github/workflows/superset-frontend.yml

```diff
@@ -1,6 +1,12 @@
 name: Frontend
 
-on: [push, pull_request]
+on:
+  push:
```

Comment:
oh nice I didn't know you could do this! GH actions ftw!

Reply:
My original GA implementation actually had some path ignores, but was [removed](https://github.com/apache/incubator-superset/pull/9538) out of abundance of caution.

Reply:
I think the main issue is that the check cannot be conditionally run _and_ required. We'd have to make all these checks not required and rely on reviewers/mergers to use their best judgement when merging.